### PR TITLE
Assignment 1 Folder and ReadMe

### PR DESCRIPTION
**Pull request by clarkis117**
PR for assignment 1 user folder and readme

### CODE REVIEW

File: Assignment_1/Clark/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Assignment 1
+Joshua Ryan Clark
+
+Github: clarkis117
+
+https://github.com/clarkis117
```
Comment (desoukya) [inline Assignment_1/Clark/README.md]: dude, what is this?

**Review by desoukya — CHANGES_REQUESTED** [review]

File: Assignment_1/Clark/README.md (hunk 1 above)
Comment (clarkis117) [reply Assignment_1/Clark/README.md]: A link to the GitHub profile 💯

Comment (clarkis117) [reply Assignment_1/Clark/README.md]: :shipit:

**Comment by clarkis117** [comment]
:shipit:

**Pull request merged**